source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -164,9 +274,83 @@ provision the following networking API entities, represented in yaml:
       - subnet_id: "{{ ipv6_subnet.id }}"
 ~~~~~
 
+### Configuring stateless DHCPv6 on the subnet
+
+To configure stateless Dhcpv6 on the subnet, the user should provision
+the following yaml based ansible playbook snippet:
+
+~~~~~
+
+- name: create a network
+  os_network:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: netv6
+  register: ipv6_network
+
+- name: attach a subnet to the logical network
+  os_subnet:
+    cloud: "{{ cloud_name }}"
+    state: present
+    network_name: netv6
+    name: subnet_ipv6
+    ip_version: 6
+    cidr: bef0:1234:a890:5678::/64
+    gateway_ip: bef0:1234:a890:5678::1
+    ipv6_address_mode: dhcpv6_stateless
+  register: dhcpv6_subnet
+
+- name: create a logical switch port
+  os_port:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: lport1
+    network: "{{ ipv6_network.id }}"
+    fixed_ips:
+      - subnet_id: "{{ dhcpv6_subnet.id }}"
+
+- name: create a router
+  os_router:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: router0
+    interfaces:
+      - "{{ dhcpv6_subnet.id }}"
+
+~~~~~
+
+## UI considerations
+
+Having dynamic address assignment in EL8 requires stateless dhcpv6 subnet
+configuration, which in turn requires an OVN logical router, in order for
+the RA generation logical flows to be installed.
+
+It is outside the scope of this feature to implement the UI changes that allow
+logical routers to be provisioned, and the REST API - or its ansible wrapper
+should be used instead. An ansible example is available
+[here](#configuring-stateless-dhcpv6-on-the-subnet).
+
 ## Testing
 
-Two scenarios are recommended:
-  - single IPv6 subnet
-  - multiple IPv6 subnets connected by router
+Two scenarios are recommended, each with two different configurations, leading
+to four different tests:
+  - single IPv6 subnet using dhcpv6-stateful configuration
```

Comment:
Need router(in rhel8 guest)?

Reply:
network prefixes are configured from RA messages, which are sent by routers. Hence, whenever you need stateless dhcpv6 config, you need a router.

since in el8 the dhclient configures a /128 prefix, and relies on the prefix length advertised in the RA messages.

Reply:
Ok, so need to add that connected by router in this sentence?

Reply:
Is the sentence in https://github.com/oVirt/ovirt-site/pull/1951/files#diff-d58a2eb312890f2afd7a2b17f2463035R437 enough?

If not, I can refer the router in his one

Reply:
Yes, enough